dev/integration_tests/android_semantics_testing/test_driver/main_test.dart

```diff
@@ -168,7 +168,7 @@ void main() {
               AndroidSemanticsAction.click,
             ],
             // We can't predict the a11y focus when the screen changes.
-            ignoredActions: ignoredAccessibilityFocusActions
+            ignoredActions: ignoredAccessibilityFocusActions,
```

Comment:
Do we have an issue filled for this? Is it something that could be fixable eventually? Would be good to link here.

Reply:
I don't think this can be fixable, Android does not guarantee accessibility focus in talkback.